modules/mobile/app/controllers/mobile/v0/facility_eligibility_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class FacilityEligibilityController < ApplicationController
+      DEFAULT_PAGE_NUMBER = 1
+      DEFAULT_PAGE_SIZE = 3
+      SERVICE_TYPES = %w[primaryCare nutrition podiatry optometry audiology].freeze
+
+      def index
+        require_params
+        unless SERVICE_TYPES.include?(params[:serviceType])
```

Comment:
Added this because the upstream services was returning 500 error when it was a bad service type